tests/rewrite_fn_ptr_eq/main.c

```diff
@@ -78,4 +80,31 @@ Test(rewrite_fn_ptr_eq, main) {
     if (y || !fn) { }
     // REWRITER: if (x && IA2_ADDR(fn) && y) { }
     if (x && fn && y) { }
+
+    // REWRITER: fn = (typeof(fn)) { NULL };
+    fn = NULL;
```

Comment:
The output is totatlly broken rn but that's a good catch about custom `NULL` macros. What I had in mind would've generated `(typeof(f)) { NULL }` which wouldn't be the best thing to do. None of this is developer-facing though so I might just have all cases use `0` instead of the corresponding macros.